drivers/hmis/app/models/hmis/entity_access_loader_factory.rb

```diff
@@ -79,6 +79,8 @@ def resolve_association(entity, safety:, &block)
       else
         block.call(entity, :project)
       end
+    when Hmis::Hud::Event
+      block.call(entity, :enrollment)
     else
       resolve_through_project(entity, &block)
     end
```

Comment:
OK, I see. This type of case would also be needed for Hmis::Hud::Assessment as well.

I think instead of this, the correct approach that Theron introduced is to `include ::Hmis::Hud::Concerns::ClientProjectEnrollmentRelated` in the `Hmis::Hud::Assessment` and `Hmis::Hud::Event` model. That concern adds a relationship to project that goes through a view which includes wip enrollments, so all projects should be resolvable.

Reply:
I can make this change and merge since you'll be offline tm